x-pack/plugins/cases/server/client/sub_cases/client.ts

```diff
@@ -105,16 +105,17 @@ async function deleteSubCase(ids: string[], clientArgs: CasesClientArgs): Promis
 
     await userActionService.bulkCreate({
       soClient,
-      actions: ids.map((id) =>
+      actions: subCases.saved_objects.map((subCase) =>
```

Comment:
Iterating over the entities themselves so that we can grab the owner when creating the user action.